portal-api/api-tests/v1/gef/facilities.api-test.js

```diff
@@ -39,6 +39,11 @@ describe(baseUrl, () => {
   beforeAll(async () => {
     testUsers = await testUserCache.initialise(app);
     aMaker = testUsers().withRole(MAKER).one();
+  });
```

Comment:
Do we want to add any specific facility end date tests here, such as checking we get a validation error if we post without FED to a deal on V1?

Reply:
I've put them in deal versioning so I'm not tampering with the process.env here - does that sound ok?